modules/programs/autorandr.nix

```diff
@@ -79,6 +79,53 @@ let
         default = null;
         example = "left";
       };
+
+      transform = with types;
+        let
+          transformType = let
+            list3 = elemType: addCheck (listOf elemType) (l: length l == 3);
+          in uniq (list3 (list3 float));
+        in
+        mkOption {
+          type = nullOr transformType;
+          description  = ''
```

Comment:
Did you have a look at the option description in the generated manual? I think you might want to… 🙂

Reply:
Might be best to simply refer to the xrandr man page.

Reply:
@rycee PTAL.

```
       programs.autorandr.profiles.<name>.config.<name>.transform
           Refer to man 1 xrandr for the documentation of the transform matrix.

           Type: null or 3×3 matrix of floating point numbers

           Default: null

           Example:

               [
                 [ 0.6 0.0 0.0 ]
                 [ 0.0 0.6 0.0 ]
                 [ 0.0 0.0 1.0 ]
               ];

           Declared by:
               <home-manager/modules/programs/autorandr.nix>
```

Reply:
That's nicer! :+1:

You can still improve it by using the docbook method of referencing man pages:

```xml
Refer to 
<citerefentry>
  <refentrytitle>xrandr</refentrytitle>
  <manvolnum>1</manvolnum>
</citerefentry>
for the documentation of the transform matrix.
```

Reply:
Oh and can remove the final `;` in the example to clean it up a little 🙂

Reply:
done.

```
       programs.autorandr.profiles.<name>.config.<name>.transform
           Refer to xrandr(1) for the documentation of the transform matrix.

           Type: null or 3×3 matrix of floating point numbers

           Default: null

           Example:

               [
                 [ 0.6 0.0 0.0 ]
                 [ 0.0 0.6 0.0 ]
                 [ 0.0 0.0 1.0 ]
               ]

           Declared by:
               <home-manager/modules/programs/autorandr.nix>

```